src/components/ReportActionItem/MoneyRequestPreview/index.tsx

```diff
@@ -0,0 +1,42 @@
+// We should not render the component if there is no iouReport and it's not a split.
+// Moved outside of the component scope to allow memoization of values later.
+import lodashIsEmpty from 'lodash/isEmpty';
+import React from 'react';
+import {withOnyx} from 'react-native-onyx';
+import * as ReportActionsUtils from '@libs/ReportActionsUtils';
+import ONYXKEYS from '@src/ONYXKEYS';
+import MoneyRequestPreviewContent from './MoneyRequestPreviewContent';
+import type {MoneyRequestPreviewOnyxProps, MoneyRequestPreviewProps} from './types';
+
+// We should not render the component if there is no iouReport and it's not a split.
+// Moved outside of the component scope to allow for easier use of hooks in the main component.
+function MoneyRequestPreview(props: MoneyRequestPreviewProps) {
+    // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
I think we should move this comment down to where the logic it's referring to is, since this isn't the doc for the method. If we want to make it the doc of the method, let' use `/** */`
```suggestion
function MoneyRequestPreview(props: MoneyRequestPreviewProps) {
    // We should not render the component if there is no iouReport and it's not a split.
    // Moved outside of the component scope to allow for easier use of hooks in the main component.
    // eslint-disable-next-line react/jsx-props-no-spreading
```